report_qweb_pdf_watermark/models/report.py

```diff
@@ -0,0 +1,135 @@
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from base64 import b64decode
+from io import BytesIO
+from logging import getLogger
+
+from PIL import Image
```

Comment:
Please add `PIL` and `PyPDF2` to external_dependencies, then it not must to be bellow try/except block

Reply:
Afaik, both are requirements of Odoo itself, so no need of external dependencies nor try.

Reply:
Ok, so imo it is unnecessary to check if they are available

Reply:
> Ok, so imo it is unnecessary to check if they are available

Indeed, it was an dependency on the past. But since it is a core dependency it had been removed from submodules. 
Do you want to leave an approving review?

Reply:
sure

Reply:
To complete my answer, it was changed in this pr: https://github.com/OCA/reporting-engine/pull/623